packages/conjure-client/src/fetchBridge/__tests__/nodeFetchStreamAdapter.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2019 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MediaType } from "../../httpApiBridge";
+
+export function nodeFetchStreamAdapter(
```

Comment:
this is a necessary hack since `node-fetch` explicitly makes returned streams be node streams instead of web streams